lib/_tls_wrap.js

```diff
@@ -1412,7 +1412,7 @@ Server.prototype.addContext = function(servername, context) {
   if (!servername) {
     throw new ERR_TLS_REQUIRED_SERVER_NAME();
   }
-
+  assert(typeof context==="object" && context)
```

Comment:
We typically make use of the internal errors system for adding type checks, and use `assert()` only for internal assertions that can’t be reached from the outside.

Could you switch this so it uses `ERR_INVALID_ARG_TYPE`, like other parts of the file do?